zerver/lib/event_types.py

```diff
@@ -42,26 +42,26 @@ class Attachment(BaseModel):
 
 
 class AttachmentAddEvent(BaseEvent):
-    type: Literal["attachment"]
-    op: Literal["add"]
+    type: Literal["attachment"] = "attachment"
+    op: Literal["add"] = "add"
     attachment: Attachment
     upload_space_used: int
 
 
-class AttachmentFieldForAttachmentRemoveEvent(BaseModel):
+class IdHolder(BaseModel):
     id: int
```

Comment:
This could probably use a docstring. I'm not sure why it isn't just a normal `int` annotation?

Reply:
It's a one-field struct that holds an int.

Reply:
It's the difference between `attachment_id: 99` and `attachment: {"id": 99}`.  For some reason we chose to use the latter in our API.  I think `IdHolder` -> `IdStruct` may be an improvement here.  I don't know what to put in the docstring.

Reply:
`IdHolder` seems fine. This seems OK for a docstring.

> A simple object with a single key, an integer ID. Presumably a simple ID was not selected for extensibility or something.

Reply:
Added a comment with different wording.